spec/requests/v1.0/workflows_spec.rb

```diff
@@ -8,7 +8,7 @@
 
   let!(:template) { create(:template) }
   let(:template_id) { template.id }
-  let!(:workflows) { create_list(:workflow, 20, :template_id => template.id) }
+  let!(:workflows) { create_list(:workflow, 16, :template_id => template.id) }
```

Comment:
why 16 or 20?

Reply:
20 exceeds the test boundary due to the extra seeding one